source/behavior/spawn_rules/components/brightness_filter.json

```diff
@@ -8,12 +8,14 @@
     "min": {
       "type": "integer",
       "description": "This is the minimum light level value that allows the mob to spawn.",
-      "title": "Min"
+      "title": "Min",
+      "minimum": 0
```

Comment:
```suggestion
      "minimum": 0,
      "maximum": 0,
      "default": 0
```

Reply:
I don't think max should be 0, probably just 15 too.

Reply:
Yes my bad 😬